packages/tools/devtools/devtools-view/src/components/data-visualization/EditableView.tsx

```diff
@@ -176,21 +118,65 @@ export function EditableView(props: EditableViewProps): React.ReactElement {
 	 * @param data - The option selected from the dropdown
 	 */
 	const onOptionSelect: DropdownProps["onOptionSelect"] = (event, data) => {
-		setEditType(data.optionText ?? "undefined");
-		setEditingComponent(selectEditUI(data.optionText ?? "undefined"));
+		setEditType(data.optionText ?? "blhe");
```

Comment:
Will `data.optionText` ever be undefined? What does it mean for it to be undefined? If that is a valid case, we should probably allow `editType` to be undefined and handle that case in the component. If not, it's probably better for us to throw an error here if the data is undefined.

Reply:
I saw that earlier and added an if statement to throw an error if its undefined.